.gitignore

```diff
@@ -1,16 +0,0 @@
-.vscode
-.DS_Store
-build
-dist
-.ipynb_checkpoints/
-*.so
-*.pyc
-*.txt
-dependency_links.txt
-PKG-INFO
-SOURCES.txt
-top_level.txt
-
-sparse_dot_topn/sparse.cpp
-sparse_dot_topn.egg-info
-sparse_dot_topn/sparse_dot_topn.cpp
```

Comment:
I missed this, why did you delete .gitignore?
Could you put it back?

Reply:
I was of the impression that .gitignore is a local repository file which often differs from local machine to local machine which is why I deleted it (actually, I didn't delete it, I made it (git-)ignore itself).   In other words, the files which I ignore on my local machine could be very different from yours.   Are you sure you want it back?

Reply:
Yes please put it back, it is a common practice.

Reply:
Done